driver/controller.go

```diff
@@ -341,13 +339,20 @@ func (d *Driver) ValidateVolumeCapabilities(ctx context.Context, req *csi.Valida
 
 // ListVolumes returns a list of all requested volumes
 func (d *Driver) ListVolumes(ctx context.Context, req *csi.ListVolumesRequest) (*csi.ListVolumesResponse, error) {
+	if req.StartingToken != "" {
+		_, err := strconv.Atoi(req.StartingToken)
+		if err != nil {
+			return nil, status.Errorf(codes.Aborted, "starting_token is invalid: %s", err)
+		}
+	}
```

Comment:
Specification is here, search for `starting_token`. 
https://github.com/container-storage-interface/spec/blob/master/spec.md

This line is here due to csi-sanity. But maybe the test isn't even correct:
https://github.com/kubernetes-csi/csi-test/issues/222

And see also:
https://github.com/kubernetes-csi/csi-test/pull/205
https://github.com/kubernetes-csi/csi-test/pull/226

Maybe we should always abort when the token is set as described "ListVolumes Errors" on the spec page.

Reply:
> Maybe we should always abort when the token is set as described "ListVolumes Errors" on the spec page.

Yeah, this makes sense I guess. If it's not easily possible (because of the tests), I would argue that a comment would be nice (`starting_token is for pagination, which we don't use, but CSI needs it for tests`).

Reply:
Implemented as discussed.